auth/v1.js

```diff
@@ -134,14 +128,14 @@ api.declare({
 }, async function(req, res) {
   let clientId = req.params.clientId;
 
-  // Load client
-  let client = await this.Client.load({clientId}, true);
+  // Load client from the resolver's cache
+  let client = this.resolver.jsonClient(clientId);
```

Comment:
And yeah, I realize we talked about doing prefix matching because we had all clients in memory... How naive are we :)

He he, actually both might work, but slow and safe might be best... Otherwise the compromise is also acceptable...